engine/src/main/scala/org/scalarules/dsl/core/projections/ProjectableFields.scala

```diff
@@ -47,9 +47,61 @@ trait ProjectableFields[C] {
     * @return a DslEvaluation of the same type as the projected field. This evaluation will at runtime
     *         provide the value of the projected field of the accompanying Fact.
     */
-  protected def projectField[F](f: C => F): DslEvaluation[F] = DslEvaluation(
+  def projectField[F](f: C => F): DslEvaluation[F] = DslEvaluation(
```

Comment:
Is there a reason why you made this function public instead of protected? The `projectField`-function is only intended to be used by the extending classes (and you don't want it to show up in code-completion in the DSL).